web/packages/teleport/src/stores/storeUserContext.ts

```diff
@@ -157,14 +157,22 @@ export default class StoreUserContext extends Store<UserContext> {
   // has access to download either teleport binaries or the license.
   // Since the page is used to download both of them, having access to one
   // is enough to show access this page.
-  // This page is only available for `dashboards`.
+  // This page is only available for `dashboards` and cloud customers.
   hasDownloadCenterListAccess() {
     return (
-      cfg.isDashboard &&
-      (this.state.acl.license.read || this.state.acl.download.list)
+      cfg.isCloud ||
+      (cfg.isDashboard &&
+        (this.state.acl.license.read || this.state.acl.download.list))
```

Comment:
just wanted to double check if it was intended. if the flag is cloud, then we _dont_ do RBAC checks? (i'm also curious why)

Reply:
Yeah, we will display this tab for all Cloud users.

As for the why, I don't know the exact reasoning (cc @xinding33), but I imagine that:

1 - most users should need `tsh`, so this makes it easier for them to find the correct binary without having to go to our website, and
2 - this way we don't have to add permissions to the `download` resource in the default rules, which is good since these roles would mean nothing for on-prem clusters.

I talk about the alternatives in the `Show Condition` section on [this comment](https://github.com/gravitational/cloud/issues/6667#issuecomment-1836619626) if you wanna take a look.